.drone.star

```diff
@@ -673,19 +668,18 @@ def notify():
         "kind": "pipeline",
         "type": "docker",
         "name": "chat-notifications",
-        "clone": {
-            "disable": True,
-        },
         "steps": [
             {
-                "name": "notify-rocketchat",
-                "image": PLUGINS_SLACK,
-                "settings": {
-                    "webhook": {
-                        "from_secret": config["rocketchat"]["from_secret"],
+                "name": "notify-matrix",
+                "image": OC_CI_ALPINE,
+                "environment": {
+                    "MATRIX_TOKEN": {
+                        "from_secret": "matrix_token",
                     },
-                    "channel": config["rocketchat"]["channel"],
                 },
+                "commands": [
+                    "bash /drone/src/tests/drone/notification.sh",
+                ],
             },
         ],
         "trigger": {
```

Comment:
status trigger should be similar to ocis (i.e. send notification of failed builds only for PRs and send both for tag and cron events)
https://github.com/owncloud/ocis/blob/2f9f2a6fdc0df0d164cb6df4656ec31fd66cf22b/.drone.star#L2347-L2349